src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * Association class for tags. Tracks tags that are added or removed.
+ */
+public class Tagging {
+    private boolean isAddition;
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.